harvardcards/static/js/modules/collections-index.js

```diff
@@ -1,11 +1,30 @@
-define(['jquery', 'components/slider/CollectionSlider', 'utils/utils'], function($, CollectionSlider, utils) {
+define(['jquery', 'components/slider/CollectionSlider', 'utils/utils', 'jqueryui'], function($, CollectionSlider, utils) {
 	return {
 		initModule: function(){
-			$('.slider').each(function(index, el) {
-				var slider = new CollectionSlider(el);
-			});
 
-			utils.setupConfirm();
+
+            $('.courseHeader').click(function() {
+                var el = this;
+                $(this).next().toggle('slow', function(){
+                    var collection_id = $(el).data('collection-id');
+                    if($(el).next().is(":visible")){
```

Comment:
Suggestion: instead of repeatedly calling `$(this).next()` or `$(el).next()`, why not capture that element in a variable and give it in an intention-revealing name. For example:

```javascript
var $courseBody = $(this).next();
...
$courseBody.toggle('slow', ...)
...
if($courseBody.is(":visible") { ... }
```

That reduces the mental overhead when reading this code of having to remember what $(el).next() means each time.